README.rst

```diff
@@ -90,6 +77,14 @@ Astropy is licensed under a 3-clause BSD style license - see the
 `LICENSE.rst <LICENSE.rst>`_ file.
 
 
+.. |Astropy Logo| image:: https://github.com/astropy/repo_stats/blob/main/dashboard_template/astropy_banner_light.svg
+    :target: https://www.astropy.org/
+    :alt: Astropy
+
+.. |User Stats| image:: https://github.com/astropy/repo_stats/blob/cache/cache/astropy_user_stats_dark.png
```

Comment:
I don't understand why logo is light but the user stats is dark here.

Reply:
Doesn't look too good in light mode. I think we need versions of both images that would look okay in either. Also the logo can be smaller. Thanks.

![Screenshot 2024-08-16 095814](https://github.com/user-attachments/assets/cfaef121-d2b2-4265-ad41-082c054932da)

Reply:
the logo file has a transparent background; maybe it was named 'light' because the text is white. 

The logo image is now the gray text version from the docs files - https://github.com/astropy/astropy/blob/main/docs/_static/astropy_banner.svg 

I'd prefer not to resize that file and have a second version of it floating around, so I think the size is ok.

Example:
https://github.com/jeffjennings/astropy/tree/readme_user_dashboard?tab=readme-ov-file